frontend/apps/crates/entry/jig/edit/src/edit/sidebar/module/state.rs

```diff
@@ -45,6 +45,26 @@ impl State {
             Some(module) => module.kind.as_str(),
         }
     }
+
+    pub fn can_add(&self) -> bool {
```

Comment:
I'm not convinced that State is the best place for this logic. Alternatively it could probably go in `module/dom.rs`

Reply:
That makes sense, you're welcome to do so